operator/controllers/manifest_controller_test.go

```diff
@@ -299,7 +302,7 @@ var _ = Describe("given manifest with a helm repo", Ordered, func() {
 			Entry("when manifestCR contains invalid Kustomize specification", createManifestWithInvalidKustomize()),
 			Entry("when manifestCR contains a valid helm repo", createManifestWithHelmRepo()),
 			Entry("when two manifestCRs contain valid OCI Image specifications", createManifestWithOCI()),
-			Entry("when manifestCR contains invalid OCI Image specification", createManifestWithInvalidOCI()),
+
```

Comment:
this should not result in race condition, since tests are `Ordered`

Reply:
It does because you use the same registry under testing.com and you dont change the layers.